### ISSUE TITLE
Accessibility - iframe (Chart.js) has no title

### ISSUE THREAD

**Issue by shaneporter** [datasciencecampus/sdg-indicators#338]
Fails Google Accessibility Tools check:

<img width="674" alt="screen shot 2017-09-26 at 09 41 43" src="https://user-images.githubusercontent.com/2493614/30850968-300c86e4-a29f-11e7-9f98-b984471c640a.png">
